src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/EventHandler.java

```diff
@@ -210,6 +223,14 @@ private boolean vmStartEvent(Event event)  {
         return stopOnVMStart;
     }
 
+/*[IF CRIU_SUPPORT]*/
+    private boolean vmRestoreEvent(Event event) {
+        VMRestoreEvent re = (VMRestoreEvent)event;
+        notifier.vmRestoreEvent(re);
+        return stopOnVMStart;
+    }
```

Comment:
This can be identical to newer versions:
```java
    private boolean vmRestoreEvent(VMRestoreEvent restoreEvent) {
        notifier.vmRestoreEvent(restoreEvent);
        return stopOnVMStart;
    }
```

Reply:
Updated